### PR TITLE
chore: bump alloy and rm `EvmOverrides`

### PR DESCRIPTION
**Pull request by tcoratger**

### CODE REVIEW

File: crates/revm/src/state_change.rs [hunk 1]
```diff
@@ -319,14 +319,14 @@ where
         let mut source_address = Address::ZERO;
         data.copy_to_slice(source_address.as_mut_slice());
 
-        let mut validator_public_key = FixedBytes::<48>::ZERO;
-        data.copy_to_slice(validator_public_key.as_mut_slice());
+        let mut validator_pubkey = FixedBytes::<48>::ZERO;
+        data.copy_to_slice(validator_pubkey.as_mut_slice());
```
Comment (emhane) [inline crates/revm/src/state_change.rs:323]: unnecessary to change variable name, pls revert

**Review by emhane — APPROVED** [review]
lgtm except unnecessary name change

File: crates/revm/src/state_change.rs (hunk 1 above)
Comment (tcoratger) [reply crates/revm/src/state_change.rs:323]: In the next line, in `WithdrawalRequest`, `validator_public_key` changed to `validator_pubkey` in recent alloy. So if I revert this variable name, this means:

```rust
withdrawal_requests.push(Request::WithdrawalRequest(WithdrawalRequest {
            source_address,
            validator_pubkey: validator_public_key,
            amount,
        }));
```

More verbose and complex to read no? Do you still prefer this?

Comment (emhane) [reply crates/revm/src/state_change.rs:323]: ah ok I see, makes sense then

File: Cargo.toml [hunk 2]
```diff
@@ -344,7 +344,7 @@ revm = { version = "9.0.0", features = [
 revm-primitives = { version = "4.0.0", features = [
     "std",
 ], default-features = false }
-revm-inspectors = { git = "https://github.com/paradigmxyz/revm-inspectors", rev = "52f632e" }
+revm-inspectors = { git = "https://github.com/paradigmxyz/revm-inspectors", rev = "6bbf303" }
```
Comment (DaniPopes) [inline Cargo.toml:347]:
```suggestion
revm-inspectors = { git = "https://github.com/paradigmxyz/revm-inspectors", rev = "79774a6" }
```

**Review by DaniPopes — APPROVED** [review]

**Pull request merged**